framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -161,6 +161,9 @@
      */
     private boolean shouldDefaultTypeVarLocals;
 
+    /** Should the type be calculated as a type declaration? */
```

Comment:
Some more documentation here would be helpful to understand what `the type` this is referring to.